websites/D/Dash Radio/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "apprehends",
+    "id": "202236922217758721"
+  },
+  "service": "Dash Radio",
+  "description": {
+    "en": "Dash Radio is a premium radio website without the ads. Available on over 50 platforms. (IN ORDER FOR THIS TO WORK, TEXT MUST BE STATIC NOT SCROLLING)"
+  },
```

Comment:
Use their official description:"
100% Free Music. More Variety. Zero Commercials. The Best DJ's & Stations. No BS

Reply:
Also remove the disclaimer. You should make it work either way, you can probs just read the elements or through something else?

Reply:
I attempted to make it work but it just didn't want to read the span element for some reason. It's just printing as null